src/main/java/com/example/car/sharing/controller/AuthenticationController.java

```diff
@@ -22,11 +26,14 @@ public class AuthenticationController {
     private final AuthenticationService authenticationService;
 
     @PostMapping("/login")
+    @Operation(summary = "Login user", description = "Login user with email and password, "
+            + "get JWT tokens")
```

Comment:
maybe u should describe what specifically do you get and how not just get jwt token